public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -104,6 +104,11 @@ export class LokiDatasource
   }
 
   getLogsVolumeDataProvider(request: DataQueryRequest<LokiQuery>): Observable<DataQueryResponse> | undefined {
+    // if there are any stream-queries, we are not creating a log-volume-histogram
```

Comment:
Hmmm, I understand that stream is experimental feature but I kind of don't like how it affects everything else. E.g. - shouldn't it hide log volumes only for stream queries, not all queries. Also - non related to this PR, but related to streams - why when I run stream query, I don't see results of my range/instant query. It shouldn't affect it.

Reply:
good point, i will try to change how this is implemented.